ExcelReader/Services/FileUploadService.cs

```diff
@@ -10,11 +10,9 @@ namespace ExcelReader.Services
     public interface IFileUploadService
```

Comment:
It's better to create separate file for interface.

Reply:
this was used when fileUpload was being handled by the same app. now this task is managed by separate web api but i left this here in case we want to have both options. should i will remove this now on your request